scss/6-components/_forms.scss

```diff
@@ -126,27 +126,31 @@
 //
 // Modifier: ~2 characters wide
 //
+.cads-input--two-chars,
```

Comment:
If we wanted to avoid the new class names we could have a mapping in the component from symbol to class name:

```
{
  :two_chars: "2ch",
  :four_chars: "4ch"
}[width]
```

Reply:
To clarify: the new class names are clearer so happy to migrate over to them if we want 👍